rules/util.js

```diff
@@ -134,6 +134,8 @@ const isString = node =>
   (node.type === 'Literal' && typeof node.value === 'string') ||
   node.type === 'TemplateLiteral';
 
+const hasExpressions = node => !!(node.expressions && node.expressions.length);
```

Comment:
The double `!!` was a little unclear reading at first. May I suggest?

```js
const hasExpressions = node => node.expressions && node.expressions.length > 0;
```

That way, the function is always returning a boolean without the extra type cast.

Reply:
Thanks for the tip!

Reply:
I think that my thinking was for if `node.expressions`  was not defined, the entire statement would evaluate to `undefined` (instead of an actual Boolean). Doesn't matter though because `undefined` is falsey, so I'll make that change to the test against `node.expressions.length` 👍